### ISSUE TITLE
Неузгоджена кольорова палітра в рішеннях HP HMI від різних брендів

### ISSUE THREAD

**Issue by pupenasan** [pupenasan/hmi#6]
Не зважаючи на те, що у багатьох брендах з'явилися нові підходи при побудові високоефективних HMI питання використання кольорів до сих пір не вирішено. Якщо раніше звучало питання як позначати клапан закритий/відкритий "червоний" або "зелений" або навпаки, то тепер те саме стосується світло-сірого(білого) і темного-сірого (чорного). На жаль (або на щастя) в стандарті ISA-101 про конкретні кольори нічого не сказано, так що у випадку використання на одному підприємстві різних платформ проблема заплутаності залишається. 

Ось, як бачить використання кольорів наприклад [Роквел](https://literature.rockwellautomation.com/idc/groups/literature/documents/wp/proces-wp023_-en-p.pdf)
![зображення](https://user-images.githubusercontent.com/25325920/161487566-b00ba3c2-ed93-4163-b8d7-18bb25fae459.png)

А ось [Wonderware](https://www.youtube.com/watch?v=DajpD5Xo3_U) думає навпаки 
![зображення](https://user-images.githubusercontent.com/25325920/161488000-91c132df-8425-46ab-9100-0178174cba6f.png)

**Comment by YuriyKuzmich** [comment]
Ось приблизно так бачить High Performance HMI ABB в одному зі своїх продуктів:
![image](https://user-images.githubusercontent.com/5729840/161542718-62df10bd-63ab-4493-827b-db911bbb0380.png)

**Comment by pupenasan** [comment]
> Ось приблизно так бачить High Performance HMI ABB в одному зі своїх продуктів:

тобто блакитний це типу ON?

**Comment by YuriyKuzmich** [comment]
Так. Але навіть всередині одного продукту є різні кольорові рішення (графічні теми). В яких загальний концепт зберігається, а кольори міняються. Наприклад:
![image](https://user-images.githubusercontent.com/5729840/161543342-3badc884-ecb2-4886-8f42-daecc6ab37db.png)

**Comment by pupenasan** [comment]
Дякую. Зрозуміло.
